src/app/public/modules/input-box/input-box.component.scss

```diff
@@ -102,7 +102,7 @@ sky-input-box {
         0
         0
         $sky-theme-modern-box-border-radius-default;
-      color: $sky-theme-modern-font-field-label-color;
+      color: $sky-theme-modern-font-data-label-color;
```

Comment:
☝️ Unrelated to this PR, but needed to be changed after the dependency upgrade. The field label color was removed intentionally in favor of `$sky-theme-modern-font-data-label-color`.